src/low_pass_filter.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file low_pass_filter.cpp
+ * @author Lucien C. Peach (peach@seas.upenn.edu)
+ * @brief Implementation of a discrete-time low-pass filter class.
+ * @date 9/10/2022
+ * 
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#include "kodlab_mjbots_sdk/low_pass_filter.h"
+#include <math.h>
+#include <Eigen/Geometry>
```

Comment:
**[General]** Include order should follow [the style guide](https://drake.mit.edu/styleguide/cppguide.html#Names_and_Order_of_Includes).
```suggestion
#include "kodlab_mjbots_sdk/low_pass_filter.h"

#include <math.h>

#include <Eigen/Geometry>
```